sdk/metric/example_test.go

```diff
@@ -242,6 +242,31 @@ func ExampleNewView_exponentialHistogram() {
 	)
 }
 
+func ExampleNewView_exemplarreservoirproviderselector() {
+	// Create a view that makes all metrics use a different exemplar reservoir.
+	view := metric.NewView(
+		metric.Instrument{Name: "*"},
+		metric.Stream{
+			ExemplarReservoirProviderSelector: func(agg metric.Aggregation) exemplar.ReservoirProvider {
+				// This example uses a fixed-size reservoir with a size of 10
+				// for explicit bucket histograms instead of the default
+				// bucket-aligned reservoir.
+				if _, ok := agg.(metric.AggregationExplicitBucketHistogram); ok {
```

Comment:
This is the intention of the specification -

You should be able to react to the actual buckets chosen by having access to the explicit bucket aggregation config.